src/Raven.Studio/typescript/viewmodels/database/settings/studioConfiguration.ts

```diff
@@ -3,6 +3,7 @@ import getStudioConfigurationCommand = require("commands/resources/getStudioConf
 import databaseStudioConfigurationModel = require("models/database/settings/databaseStudioConfigurationModel");
 import eventsCollector = require("common/eventsCollector");
 import saveStudioConfigurationCommand = require("commands/resources/saveStudioConfigurationCommand");
+import pwaInstaller = require("../../../common/pwaInstaller");
```

Comment:
let's avoid relative paths - r.js has issues with packing this properly

Reply:
Typescript did that for me automatically. I'll fix it.